d2l-table-style.html

```diff
@@ -236,6 +236,151 @@
 			[dir="rtl"] d2l-table[no-column-border] d2l-tbody > d2l-tr > .d2l-table-cell-first {
 				border-right: var(--d2l-table-border);
 			}
+
+			[sticky-headers] {
+				position: relative;
+				overflow: hidden;
+			}
+
+			[sticky-headers] tr[header] th {
```

Comment:
It looks like consumers need to put a `header` attribute on the header they want to be sticky... would it be possible to just treat everything inside the `<thead>` element as the header instead?

Reply:
the `[header]` attribute is used as an alternative to `thead` headers so that a header can be inlined within a table (manage grades page uses this I think)

If sticky headers are used, this attribute won't be necessary.

Reply:
I see, so it's purely to get the header styles applied to a non-header row. Agreed then, sticky-headers shouldn't need it.

Reply:
In the Enter Grades Page (user_list_view), the table has no `thead` and the only way to know if a row is a header is with that attribute. I'm happy to add `thead` as another selector but I need `[header]` for that page at the very least.

Reply:
The enter grades page should be changed so that the inline `header` attribute is removed, otherwise there will be multiple sticky headers

Reply:
Can we fix the grades page (and really the legacy table component it's using) to use `thead`s? We'll need to be cognizant of any CSS it has for pre-Daylight, which I'm guessing is why those `header` attributes are on there? But yeah... as Anthony said we probably don't want to end up with multiple sticky headers anyway.